nephio/core/configsync/configsync.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: configmanagement.gke.io/v1
 kind: ConfigManagement
-metadata: # kpt-merge: /config-management
```

Comment:
what is the reason for removing the text after `#`?

Reply:
I'm pretty sure we don't need it.
https://kpt.dev/reference/cli/pkg/update/?id=resource-identity
I'm guessing the orig pkg was created using kpt but as it stands we don't need to customize the pkg further.
@johnbelamaric @henderiw could you confirm?